### ISSUE TITLE
Photoswipe missing alt attribute WCAG 2.0 failure 

### ISSUE THREAD

**Issue by shadtek** [woocommerce/woocommerce#25233]
**Describe the bug**
Images in the Photoswipe gallery are not programmed to have an alt attribute.
1.1.1 Non-text Content Level A
The alt for images is important for WCAG 2.0 see the below 3 examples.

https://www.w3.org/TR/WCAG20-TECHS/F38
Failure of Success Criterion 1.1.1 due to omitting the alt-attribute for non-text content used for decorative purposes only in HTML

https://www.w3.org/TR/WCAG20-TECHS/F65
Failure of Success Criterion 1.1.1 due to omitting the alt attribute on img elements, area elements, and input elements of type "image"

https://www.w3.org/TR/WCAG20-TECHS/H37
Using alt attributes on img elements

**To Reproduce**
1. Go to https://themes.woocommerce.com/storefront/product/lowepro-slingshot-edge-250-aw/ and click on the magnify icon  to get the larger image of the backpack.
2. Inspect the backpack image that comes up.
3. It doesn't have an alt attribute.

**Screenshots**
Here you can see from the Storefront demo site the exact issue. No alt = WCAG 2.0 failure.
![image](https://user-images.githubusercontent.com/4212236/70829458-0cd32a00-1dab-11ea-81d8-897ebd326e52.png)

**Expected behavior**
Expected behavior is that the alt attribute shows up.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have used an example directly from the Storefront live demo.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
It's in the Storefront live demo.

</details>

**Comment by shadtek** [comment]
@tammullen Thanks for the label.

I was able to fix it this morning by registering my own scripts for photoswipe and single-product. But I think someone should take what I made and just put it into the core.

`single-product.js` I added the alt to the image item object.
```
item = {
  src  : large_image_src,
  w    : large_image_w,
  h    : large_image_h,
  title: img.attr( 'data-caption' ) ? img.attr( 'data-caption' ) : img.attr( 'title' ),
  alt: img.attr('alt') // NOTE: Added img alt for WCAG.
}
```

`photoswipe.js` after the image source is set, add the image alt
```
img.src = item.src;// + '?a=' + Math.random();\
img.alt = item.alt;// NOTE: Added alt for WCAG.
```

**Comment by tammullen** [comment]
Thanks for raising the issue @shadtek. I can reproduce it on my end.

When I do the same steps of magnifying an image on a test site there is no 'alt' attribute.
<img width="1543" alt="Screenshot_2019-12-16_at_19_24_38" src="https://user-images.githubusercontent.com/24649833/70936649-a3694a80-203a-11ea-888f-ff106bf5c415.png">

And thank you for the suggested fix!

**Comment by shadtek** [comment]
What does the error on this page mean? 
https://travis-ci.org/github/woocommerce/woocommerce/jobs/704421038
